opentelemetry-api/src/opentelemetry/metrics/label_value.py

```diff
@@ -11,3 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+
+class LabelValue:
```

Comment:
can this just be a string?

Reply:
It could be a string. However, I think if we ever needed to add additional fields to LabelValue, we could do so if we used a class without introducing breaking changes.